src/Service/Telemetry/ApplicationInsightsLoggerProvider.cs

```diff
@@ -1,85 +0,0 @@
-// Copyright (c) Microsoft Corporation.
-// Licensed under the MIT License.
-
-using System;
```

Comment:
Why dont we need the ApplicationInsightsLoggerProvider anymore?

Reply:
`AddApplicationInsights()` uses the AppInsights SDK provided `ApplicationInsightsLoggerProvider`. Source code: https://github.com/microsoft/ApplicationInsights-dotnet/blob/2.21.0/LOGGING/src/ILogger/ApplicationInsightsLoggerProvider.cs

No need to roll our own.